code/ship/ship.cpp

```diff
@@ -12583,6 +12560,10 @@ int wing_lookup(const char *name)
  */
 static int ship_info_lookup_sub(const char *token)
 {
+	// bogus
+	if (token == nullptr)
```

Comment:
If this is not a valid input then maybe it could be covered by an Assertion instead?

Reply:
I did this to match `weapon_info_lookup`.